src/components/Navbar/SideWidgets/SpeedGraph.vue

```diff
@@ -23,21 +25,22 @@ const chartOptions: ApexOptions = {
       enabled: false
     }
   },
-  colors: [theme.current.value.colors.upload, theme.current.value.colors.download],
+  colors: ['#f38a09', '#f38a09', theme.current.value.colors.upload, theme.current.value.colors.download],
   stroke: {
     show: true,
     curve: 'smooth',
     lineCap: 'round',
-    width: 4
+    width: 3,
```

Comment:
I tried to work on this myself and had some issues applying different widths on the lines in the graph.  Every time I did the whole graph just disappeared. I suppose this is a general issue coming from the apexcharts libary or?

Reply:
Yeah I had the same issue, some error in the SVG path pops up when using an array, even though the parsing is done correctly...